.github/workflows/python-app.yml

```diff
@@ -17,7 +17,7 @@ jobs:
     strategy:
       matrix:
         os: ["ubuntu-latest", "macos-latest", "windows-latest"]
-        python-version: ["3.7", "3.8", "3.9"]
+        python-version: ["3.8", "3.9", "3.10"]
```

Comment:
You also need to change the minimum python version from `3.7` to `3.8` in:

- `.readthedocs.yml`
- `dev-environement` and `environment` files
- `setup.py`

Reply:
Yes, I saw that you have included this in xdem. But in geoutils, we don't have any restriction on versions for Python apart from >3. I'm unsure about adding it. We cannot guarantee that geoutils will be stable with version below 3.8, but on the other hand, it does not strictly require Python>=3.8. It works with 3.7 and maybe below, but some rounding issues seem to happen...

Reply:
It's not tested for python <3.7, so we absolutely need pip/conda to solve the environment with this condition. It's a web of packages out there we depend on (and might depend on us eventually), if they all left untested versions possible in their environment, all hell would break loose and nothing would work! :exploding_head: 

People can force another version of python manually quite easily if they want to try it.

Reply:
Ok, done in 8946ae7